### PR TITLE
immune cell survival update to include extent tumor resect + lgg group

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Adding extent of tumor resection and lgg grouping per @adamcresnick

#### What was your approach?
Same as in #1342 

#### What GitHub issue does your pull request address?
NA

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
Removed subtype analysis from here because of #1332 and that this PR only included RNA samples.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What is your summary of the results?
pretty much the same as before, except PDL1 expression is not significant across the entire cohort

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/survival-analysis/survival-analysis_immune.Rmd [hunk 1]
```diff
@@ -99,19 +99,38 @@ quantiseq <- read_rds(
...
     select(Kids_First_Participant_ID, 
            Kids_First_Biospecimen_ID, 
            cancer_group, 
-           molecular_subtype, 
+           molecular_subtype,
+           extent_of_tumor_resection,
            OS_status, 
            OS_days, 
            PFS_days) %>%
     distinct()
```
Comment (sjspielman) [inline analyses/survival-analysis/survival-analysis_immune.Rmd:124]: I have similar comments here that I put in the analogous code chunk in #1342

Comment (jharenza) [reply analyses/survival-analysis/survival-analysis_immune.Rmd:124]: updated the recode for these pts

**Review by sjspielman — APPROVED** [review]
👍

**Comment by sjspielman** [comment]
I'm going to go ahead and merge this in since it's passed CI and been approved!

**Pull request merged**